Automattic/wp-calypso
# Updated treatment as the default upsell plan upgrade

#### Proposed Changes

We decided to move forward with the treatment screen as the default for upsell plan upgrade
Additional context: p1674227472157389/1673857251.106749-slack-C03TDMUSUSG

#### Testing Instructions



* After buying a Premium plan for a site you should see the treatment as the only option possible
* You will see it at this URL `http://calypso.localhost:3000/checkout/onemoresimplesite.wordpress.com/offer-plan-upgrade/business-monthly/20059396`

<img width="1048" alt="image" src="https://user-images.githubusercontent.com/1044309/213784589-5626f529-3797-4e4b-ab0c-1e367c7fb77f.png">
 

#### Pre-merge Checklist





Related to #
